### PR TITLE
Making HDInsight public

### PR DESCRIPTION
**Pull request by shefaliv**
Copying over HDI changes from the private repo

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@shefaliv__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (shvohra). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/ResourceManagement/HDInsightJob/HDInsightJob/app.config [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (yugangw-msft) [inline src/ResourceManagement/HDInsightJob/HDInsightJob/app.config]: remove this

File: src/ResourceManagement/HDInsightJob/HDInsightJob/Microsoft.Azure.Management.HDInsight.Job.nuspec [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>The HDInsight management client.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags>Microsoft "Microsoft Azure" Azure cloud management "service management" REST HTTP client core common azureofficial windowsazureofficial</tags>
+    <references>
+      <group targetFramework="net40">
+        <reference file="Microsoft.Azure.Management.HDInsight.Job.dll" />
+      </group>
+    </references>
+  </metadata>
```
Comment (yugangw-msft) [inline src/ResourceManagement/HDInsightJob/HDInsightJob/Microsoft.Azure.Management.HDInsight.Job.nuspec]: please dependency on common like 
https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/Resource/ResourceManagement/Microsoft.Azure.Management.Resources.nuspec#L27

File: src/ResourceManagement/HDInsightJob/HDInsightJob/Microsoft.Azure.Management.HDInsight.Job.nuspec [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>The HDInsight management client.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags>Microsoft "Microsoft Azure" Azure cloud management "service management" REST HTTP client core common azureofficial windowsazureofficial</tags>
+    <references>
+      <group targetFramework="net40">
+        <reference file="Microsoft.Azure.Management.HDInsight.Job.dll" />
+      </group>
+    </references>
+	<dependencies>
```
Comment (yugangw-msft) [inline src/ResourceManagement/HDInsightJob/HDInsightJob/Microsoft.Azure.Management.HDInsight.Job.nuspec]: could you fix the indent?

**Pull request merged**